test/ryu.jl

```diff
@@ -368,6 +372,9 @@ end
 #     return fails / (fails + success)
 # end
 
+@testset "Regression" begin
```

Comment:
I don't know if a test set called "regression" with a single test in it is that useful.